src/home_robot_sim/home_robot_sim/env/habitat_imagenav_env/habitat_imagenav_env.py

```diff
@@ -0,0 +1,67 @@
+from typing import Any, Dict, cast
+
+import habitat
+import numpy as np
+from habitat.sims.habitat_simulator.actions import HabitatSimActions
+
+import home_robot
+from home_robot_sim.env.habitat_abstract_env import HabitatEnv
+
+
+MIN_DEPTH_REPLACEMENT_VALUE = 10000
+MAX_DEPTH_REPLACEMENT_VALUE = 10001
+
+
+class HabitatImageNavEnv(HabitatEnv):
```

Comment:
general comment for this file: docstrings + type hints for code going in main `src/` repositories.